src/vcpkg/dependencies.cpp

```diff
@@ -1715,43 +1705,33 @@ namespace vcpkg::Dependencies
             Util::sort_unique_erase(specs);
             m_var_provider.load_dep_info_vars(specs);
             const auto& vars = m_var_provider.get_dep_info_vars(toplevel).value_or_exit(VCPKG_LINE_INFO);
-            std::vector<const Dependency*> active_deps;
 
```

Comment:
This is semantically changing the algorithm and should not be done during a documentation pass.

We perform this sequence to ensure the `default_features` of each top level package is set to false _before_ recursing into the individual packages. Otherwise, a case like:

```
A -> B, C[core]
B -> C
```
could install the default features of `C`.